task.aspx.cs

```diff
@@ -9,9 +12,275 @@ namespace LogBook
 {
     public partial class task : System.Web.UI.Page
     {
+
+        public string FirstName { get {
+                return txtTask.Text;
+            }
+            set
+            {
+                txtTask.Text = value;
+            }
+        }
+
+
+        readonly string strCon = ConfigurationManager.ConnectionStrings["con"].ConnectionString;
+
+
         protected void Page_Load(object sender, EventArgs e)
         {
+            GridView1.DataBind();
+        }
+
+        /// <summary>Add Button Click</summary>
+        protected void BttnAdd_Click(object sender, EventArgs e)
+        {
+            
+            if (CheckIfTaskExists())
+            {
+                Response.Write("<script>alert('Task with this ID already Existed.');</script>");
+            }
+            else
+            {
+                AddNewTask();
+            }
+        }
+
+        /// <summary>Update Button Click</summary>
+        protected void BttnUpdate_Click(object sender, EventArgs e)
+        {
+            if (CheckIfTaskExists())
+            {
+                UpdateTask(); 
+            }
+            else
+            {
+                Response.Write("<script>alert('Task does not Exist.');</script>");
+            }
+        }
+
+        /// <summary>Delete Button Click</summary>
+        protected void BttnDelete_Click(object sender, EventArgs e)
+        {
+            if (CheckIfTaskExists())
+            {
+                DeleteTask();
+            }
+            else
+            {
+                Response.Write("<script>alert('Task does not Exist.');</script>");
+            }
+        }
+
+        /// <summary>Go Button Click</summary>
+        protected void LinkButton4_Click(object sender, EventArgs e)
+        {
+            GetTaskById();
+        }
+
+        ///<summary>
+        ///user defined function
+        ///</summary>
+        void GetTaskById()
+        {
+            try
+            {
+                SqlConnection con = new SqlConnection(strCon);
+                if (con.State == ConnectionState.Closed)
+                {
+                    con.Open();
+                }
+
+                SqlCommand cmd = new SqlCommand("SELECT * from task_detail_tbl where task_id='" + txtTaskId.Text.Trim() + "';", con);
+                SqlDataAdapter da = new SqlDataAdapter(cmd);
+                DataTable dt = new DataTable();
+                da.Fill(dt);
+
+                if (dt.Rows.Count >= 1)
```

Comment:
### Simpler checking

It's easier to check for Rows > 0 if you want to know if there are any values. 
```suggestion
                if (dt.Rows.Count > 0)
```